frontend/src/tests/components/Commons/ManageCowsModal.test.js

```diff
@@ -32,6 +32,22 @@ describe('ManageCowsModal', () => {
     expect(screen.getByTestId('buy-sell-cow-modal')).toBeInTheDocument();
   });
 
+  test('modal has correct buy capitalization', () => {
+    render(
+        <ManageCowsModal isOpen={true} onClose={mockOnClose} message="buy" setNumber={mockSetNumber} />
```

Comment:
Why not just do this:
```suggestion
        <ManageCowsModal isOpen={true} onClose={mockOnClose} message="Buy" setNumber={mockSetNumber} />
```

Reply:
resolving because of this:
https://github.com/ucsb-cs156-f24/proj-happycows-f24-10/pull/14#issuecomment-2480151773